File: src/main/java/com/snowofsunflower/android/ui/talker/IPromptTalker.java
package com.snowofsunflower.android.ui.talker;

/**
 * Created by zhouztashin on 2018/10/21.
 */

public interface IPromptTalker extends ITalker<IPromptTalker> {
    IPromptTalker reactor(Reactor reactor);

    IPromptTalker detail(String str);

    IPromptTalker reactText(String str);
}

File: src/main/java/com/snowofsunflower/android/ui/talker/IPromptTalker.kt
package com.snowofsunflower.android.ui.talker

import android.view.View

/**
 * Created by zhouztashin on 2018/10/21.
 * 提示对话框
 */

interface IPromptTalker : ITalker<IPromptTalker> {
    /**
     * 提示按钮响应
     */
    fun reactor(reactor: (v: View, talker: IPromptTalker) -> Unit): IPromptTalker

    /**
     * 提问详情
     */
    fun detail(str: String): IPromptTalker

    /**
     * 响应文字
     */
    fun reactText(str: String): IPromptTalker
}
